packages/mobile/src/account/saga.ts

```diff
@@ -108,8 +145,10 @@ export function* watchKycStatus() {
   try {
     while (true) {
       const kycStatus = yield take(channel)
+      Logger.debug(TAG + '@watchKycStatus', 'KYC status', kycStatus)
       if (kycStatus === undefined || Object.values(KycStatus).includes(kycStatus)) {
         yield put(updateKycStatus(kycStatus))
+        yield call(fetchFinclusiveKyc)
```

Comment:
I'd be more ok with putting some logic around persona kyc status here

Reply:
since this function deals with persona kyc status explicitly already

Reply:
done